addon/templates/components/uni-tooltip.hbs

```diff
@@ -0,0 +1,15 @@
+<span class="uni-tooltip__icon">
+    {{#if informationIcon}}
+        {{inline-svg "icons/information-icon"}}
+    {{else if questionIcon}}
+        {{inline-svg "icons/question-icon"}}
+    {{else}}
+        {{yield}}
```

Comment:
I needed that when I wanted something to open the tooltip when is hovered, by sending it in yield, it works